<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SI21xxDialog</class>
 <widget class="QDialog" name="SI21xxDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>I2C Controlled DVB-S SI21xx</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>681</width>
     <height>401</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>i2c-dev</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>441</width>
       <height>52</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>i2c-dev:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="i2c_dev">
        <property name="text">
         <string>/dev/i2c-5</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="openDevBtn">
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QLabel" name="si21_id_label">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>DVB-S SI21?? Rev ?</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Tune</string>
    </attribute>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>24</y>
       <width>649</width>
       <height>227</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Demodulation</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="modulationCB">
        <item>
         <property name="text">
          <string>BPSK</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QPSK</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="deintCB">
        <property name="text">
         <string>Bypass deinterleaver</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QCheckBox" name="descramblerCB">
        <property name="text">
         <string>Bypass descrambler</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Viterbi Code Rate</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="fecCB">
        <item>
         <property name="text">
          <string>1/2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2/3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3/4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5/6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6/7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7/8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Search All</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Search None</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="rsCB">
        <property name="text">
         <string>Bypass Reed-Solomon</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>LNB Voltage</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="lnbCB">
        <item>
         <property name="text">
          <string>13 V</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>18 V</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>LNB LO Frequency [MHz]</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="loSB">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="maximum">
         <double>10000000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Frequency [MHz]</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="freqSB">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>950.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Search range [+/-]</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QComboBox" name="csrCB">
        <item>
         <property name="text">
          <string>12 MHz (QuickLock)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6.3 MHz</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3.1 MHz</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1.6 MHz</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>800 KHz</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>400 KHz</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>200 KHz</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>100 KHz</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Symbol rate [KS/s]</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QDoubleSpinBox" name="symbolrateSB">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>1000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>45000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>1000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QPushButton" name="tuneBtn">
        <property name="text">
         <string>Tune</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Status</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>631</width>
       <height>341</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Lock/Acqusition</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>591</width>
         <height>281</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="lockstatusLabel">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Lock Status: 0x0000</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="acqusitionstatusLabel">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Acqusition Errors: 0x0080</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="agclCb">
          <property name="text">
           <string>AGC Lock Complete</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="aqfCb">
          <property name="text">
           <string>Receiver Acquisition Failed</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="celCb">
          <property name="text">
           <string>Carrier Estimation Complete</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QCheckBox" name="agcfCb">
          <property name="text">
           <string>AGC Search Failed</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="srlCb">
          <property name="text">
           <string>Symbol Rate Estimation Complete</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QCheckBox" name="cefCb">
          <property name="text">
           <string>Carrier Estimation Search Failed</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QCheckBox" name="stlCb">
          <property name="text">
           <string>Symbol Timing Lock</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QCheckBox" name="srfCb">
          <property name="text">
           <string>Symbol Rate Search Failed</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QCheckBox" name="crlCb">
          <property name="text">
           <string>Carrier Lock</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QCheckBox" name="stfCb">
          <property name="text">
           <string>Symbol Timing Search Failed</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QCheckBox" name="vtlCb">
          <property name="text">
           <string>Viterbi Lock</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QCheckBox" name="crfCb">
          <property name="text">
           <string>Carrier Search Failed</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QCheckBox" name="fslCb">
          <property name="text">
           <string>Frame Sync Lock</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QCheckBox" name="vtfCb">
          <property name="text">
           <string>Viterbi Search Failed</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QCheckBox" name="rcvlCb">
          <property name="text">
           <string>Receiver Lock</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QCheckBox" name="fsfCb">
          <property name="text">
           <string>Frame Sync Search Failed</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QCheckBox" name="bsdaCb">
          <property name="text">
           <string>Blindscan Data Ready For Host</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QCheckBox" name="bsdoCb">
          <property name="text">
           <string>Blindscan Done</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
